src/main/java/net/datafaker/transformations/Transformer.java

```diff
@@ -29,18 +32,24 @@ default String getElementSeparator() {
         return "";
     }
 
-
     default Stream<OUT> generateStream(final Schema<IN, ?> schema, long limit) {
         Item item = new Item(0);
         return Stream.generate(() -> {
             StringBuilder res = new StringBuilder();
             if (item.current == 0) {
-                res.append(getStartStream(schema)).append(getLineSeparator());
+                final String startStream = getStartStream(schema);
+                if (startStream != null) {
+                    res.append(getStartStream(schema)).append(getLineSeparator());
```

Comment:
Why bother with the intermediate store? Why not just call the method in the condition? (Esp since it's called again to append anyway)

Reply:
Or do it like the end one and use it for the append 😉

Reply:
seems forgot about this, thanks